<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gecon::GestureEventDialog</class>
 <widget class="QDialog" name="Gecon::GestureEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ON event:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="onEventGesture"/>
      </item>
      <item>
       <widget class="QComboBox" name="onEvent"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="offEventCheckbox">
        <property name="text">
         <string>OFF event:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="offEventGesture"/>
      </item>
      <item>
       <widget class="QComboBox" name="offEvent"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Gecon::GestureEventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Gecon::GestureEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
